pyproject.toml

```
@@ -31,7 +31,7 @@ coveralls = "coveralls.cli:main"
 python-coveralls = "coveralls.cli:main"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.13"
+python = ">=3.8,<3.14"
```

I recommend removing the upper limit, it will prevent this problem next year (and beyond) and also prevents us testing our projects on 3.14 now (it's already in alpha):

```suggestion
python = ">=3.8"
```

See https://iscinumpy.dev/post/bound-version-constraints/ for the long version.

---

Changed.